src/database/migrations/1620171366267-CreateSongs.ts

```diff
@@ -30,18 +30,21 @@ export class CreateSongs1620171366267 implements MigrationInterface {
           type: 'varchar',
         }
       ],
-      foreignKeys: [
-        {
-          columnNames: [ 'playlist_id' ],
-          referencedColumnNames: [ 'id' ],
-          referencedTableName: 'playlist',
-          onDelete: 'CASCADE',
-        }
-      ]
+    }));
+
+    await queryRunner.createForeignKey('songs', new TableForeignKey({
+      name: 'PlaylistIdentification',
+      columnNames: ['playlist_id'],
+      referencedColumnNames: ['id'],
+      referencedTableName: 'playlist',
+      onDelete: 'SET NULL',
```

Comment:
```suggestion
      onDelete: 'CASCADE',
```

se a playlist for excluída as música precisam ser excluídas também